living_documentation_regime/README.md

```diff
@@ -0,0 +1,358 @@
+
+Using a dedicated repository solely for documentation tickets provides multiple advantages:
+- **Streamlined Management:** This avoids cross-project conflicts and board exclusions and enables specialized templates solely for documentation purposes.
+- **Focused Access Control:** This allows a small team to manage and edit documentation without interference, maintaining high-quality content.
+- **Optimized Data Mining:** Supports easier and more efficient data extraction for feedback and review cycles through Release Notes.
+- **Implementation Reflection:** Mirrors elements from the implementation repositories, providing a high-level knowledge source that is valuable for both business and technical teams.
+- **Release Notes Integration:** Documentation can evolve based on insights from release notes, serving as a dynamic feedback loop back to the documentation repository.
+
+---
+## Living Documentation Regime Features
+
+### Data Mining from GitHub Repositories
+
+This feature allows you to define which repositories should be included in the living documentation process. By specifying repositories, you can focus on the most relevant projects for your documentation needs.
+
+- **Default Behavior**: By default, the action will include all repositories defined in the repositories input parameter. Each repository is defined with its organization name, repository name, and query labels.
+
+### Data Mining from GitHub Projects
+
+This feature allows you to define which projects should be included in the living documentation process. By specifying projects, you can focus on the most relevant projects for your documentation needs.
+
+- **Default Behavior**: By default, the action will include all projects defined in the repositories. This information is provided by the GitHub API.
+- **Non-default Example**: Use available options to customize which projects are included in the documentation.
+  - `project-state-mining: false` deactivates the mining of project state data from GitHub Projects. If set to **false**, project state data will not be included in the generated documentation and project related configuration options will be ignored. 
+  - `projects-title-filter: []` filters the repository attached projects by titles, if list is empty all projects are used.
+      ```json
+        {
+          "organization-name": "absa-group",
+          "repository-name": "living-doc-example-project",
+          "query-labels": ["feature", "bug"],
+          "projects-title-filter": ["Community Outreach Initiatives", "Health Data Analysis"]
+         }
+      ```
+
+### Living Documentation Page Generation
+
+The goal is to provide a straightforward view of all issues in a single table, making it easy to see the overall status and details of issues across repositories.
+
+- **Default Behavior**: By default, the action generates a single table that lists all issues from the defined repositories.
+
+### Structured Output
```

Comment:
```suggestion
#### Structured Output
```

Reply:
This is one of liv-doc regime features, it should have the same heading level as others features.

Reply:
I firstly did not understand your idea for having different headings for the features. But get it now and it is a good approach! See: [22593b3](https://github.com/AbsaOSS/living-doc-generator/pull/72/commits/22593b349c9ef8e9921c2ffeed07b527c8c2591b).